compiler/daml-extension/BUILD.bazel

```diff
@@ -73,6 +73,15 @@ genrule(
     tools = ["//bazel_tools/sh:mktgz"],
 )
 
+genrule(
+    name = "webview-stylesheet.css",
+    srcs = ["src/webview.css"],
+    outs = ["webview-stylesheet.css"],
+    cmd = """
+    cp "$(location src/webview.css)" "$@"
+    """,
+)
```

Comment:
That doesn't seem to work?
`bazel build //compiler/daml-extension:src/webview-stylesheet.css` gives an error, and a "runfiles" approach didn't work

Reply:
Right, the runfiles approach isn't going to work since that would require bundling `webview.css` into `damlc`, which doesn't feel like the right place for it. 

Regarding the `bazel build` error, I forgot that files in subdirectories are private by default. I think an `exports_files([...])` statement would do the trick, but 

> **Best practice:** When possible, prefer to expose a rule target rather than a source file. For example, instead of calling exports_files on a .java file, wrap the file in a non-private java_library target. Generally, rule targets should only directly reference source files that live in the same package.
> -- https://bazel.build/concepts/visibility#source-file-target-visibility

so you're doing the right thing as it is. I just wish there was a less verbose way to do it 😅